src/index.css

```diff
@@ -1,5 +1,253 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
+
+body {
+  margin: 20px;
+}
+
+.navigation-bar {
+  display: flex;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  background-color: rgb(253, 246, 246);
+  border-bottom: 1px solid gray;
+  margin-left: 4rem;
+  margin-top: 1.5rem;
+}
+
+.navigation-bar:hover {
+  border-bottom: rgb(10, 80, 172);
+  border: 1px solid rgb(10, 80, 172);
+}
+
+.main-heading {
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: rgb(6, 86, 233);
+  padding-top: 1rem;
+  padding-left: 2rem;
+}
+
+.thelinks {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  text-decoration: none;
+  padding-top: 2rem;
+}
+
+.ho a {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: normal;
+  color: rgb(26, 24, 24);
+  text-decoration: none;
+  margin: 2rem;
+}
+
+.ca a {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: normal;
+  color: rgb(118 113 113);
+  text-decoration: none;
+}
+
+.navigation-bar span {
+  width: 2.813rem;
+  height: 2.813rem;
+  font-size: 3rem;
+  color: rgb(6, 86, 233);
+  margin-left: 30rem;
+  padding-top: 1rem;
+}
+
+.form-container {
+  margin: auto;
+  padding-left: 4rem;
+  padding-bottom: 8rem;
+  background: rgb(253, 246, 246);
+}
+
+#category {
+  width: 18.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 0.813rem 0.688rem 0.3rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #888;
+  background-color: hsl(11deg 80% 38% / 8%);
+}
+
+#title {
+  width: 41.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 0.4rem 0.688rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #888;
+  background-color: hsl(11deg 80% 38% / 8%);
+}
+
+#category:hover {
+  border: 1px solid blue;
+}
+
+#title:hover {
+  border: 1px solid blue;
+}
+
+.addbtn {
+  width: 11.938rem;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+  margin-top: 1rem;
+  border-radius: 0.5rem;
+  background: rgb(6, 86, 233);
+  color: wheat;
+}
+
+.addtitle {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  color: #888;
+  padding-top: 4rem;
+}
+
+.form-container .inputt {
+  display: flex;
+}
+
+.lower-remove {
+  color: rgb(6, 86, 233);
+  display: flex;
+}
+
+.removebtn {
+  border: none;
+  color: rgb(6, 86, 233);
+  background: rgb(253, 246, 246);
+  font-size: 1rem;
+}
+
+.removebtn:hover {
+  color: rgb(55, 55, 243);
+  border: 1px solid rgb(83, 83, 211);
+}
+
+.firstpart {
+  display: flex;
```

Comment:
- [ ] There are multiple declarations of display: flex, to follow best practices of CSS - DRY(Don't Repeat Yourself) I suggest creating a class for display flex to use where needed. Check this [article](https://speckyboy.com/good-bad-css-practices/) for more explanation.

Reply:
- [ ] Kindly check for other repetitions and modify them.